### PR TITLE
feat: add cart page screen

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs 
## Screenshot
Screenshot (150)

### CODE REVIEW

**Comment by poojakelkar** [comment]
Add the total amount in bold, Otherwise great work looks good to merge

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  <link rel="stylesheet" href="https://arcui.netlify.app/styles/padding.css" />
+  <link rel="stylesheet" href="../styles/cart.css" />
+  <title>Meck Store | Cart</title>
+</head>
+
+<body>
+  <!-- Navbar -->
+  <header class="header">
+    <div class="header-wrapper mt-2 ml-4">
+      <!-- <img class="logo" src="#" alt="" /> -->
```
Comment (HarshnaNHaswani) [inline pages/cart.html:26]: remove comment

**Review by HarshnaNHaswani — APPROVED** [review]
Looks really clean, nice UI

File: pages/cart.html (hunk 1 above)
Comment (devdeadviz) [reply pages/cart.html:26]: I will add the logo of the e-commerce app here, later on, that's why they are commented.

**Review by poojakelkar — COMMENTED** [review]
Looks good to merge

**Comment by devdeadviz** [comment]
> Add the total amount in bold, Otherwise great work looks good to merge

I have done it intentionally to keep the price value smilar.

**Review by pnchinmay — APPROVED** [review]

**Pull request merged**